### ISSUE TITLE
No Longer Appears to Work 

### ISSUE THREAD

**Issue by thecodegoddess** [gilbarbara/alfred-workflows#3]
I am not longer getting any results back

```
[09:53:15.552] MDN Search[Script Filter] Queuing argument 'E'
[09:53:15.927] MDN Search[Script Filter] Queuing argument 'Ev'
[09:53:15.965] MDN Search[Script Filter] Script with argv '(null)' finished
[09:53:15.967] MDN Search[Script Filter] <?xml version="1.0"?>
<items><item uid="placeholder" arg="https://developer.mozilla.org/en-US/search?q=E" valid="yes"><title>No documents were found that matched "E".</title><subtitle>Click to see the results for yourself</subtitle><icon>icon.png</icon></item></items>
[09:53:16.222] MDN Search[Script Filter] Queuing argument 'Eve'
[09:53:16.488] MDN Search[Script Filter] Queuing argument 'Even'
[09:53:16.632] MDN Search[Script Filter] Queuing argument 'Event'
[09:53:17.115] MDN Search[Script Filter] Script with argv '(null)' finished
[09:53:17.118] MDN Search[Script Filter] <?xml version="1.0"?>
<items><item uid="placeholder" arg="https://developer.mozilla.org/en-US/search?q=Event" valid="yes"><title>No documents were found that matched "Event".</title><subtitle>Click to see the results for yourself</subtitle><icon>icon.png</icon></item></items>
[09:53:17.678] MDN Search[Script Filter] Queuing argument 'Events'
[09:53:18.144] MDN Search[Script Filter] Script with argv '(null)' finished
[09:53:18.146] MDN Search[Script Filter] <?xml version="1.0"?>
<items><item uid="placeholder" arg="https://developer.mozilla.org/en-US/search?q=Events" valid="yes"><title>No documents were found that matched "Events".</title><subtitle>Click to see the results for yourself</subtitle><icon>icon.png</icon></item></items>
```

**Comment by jljorgenson18** [comment]
Same thing here.

**Comment by gilbarbara** [comment]
It's fixed in version 1.1
